app/src/main/java/com/example/popularmoviesstage2/TrailersResponse.java

```diff
@@ -5,6 +5,8 @@
 import java.util.List;
 //This is a test comment
 
+//This is a test comment2
```

Comment:
Please remove this comment and replace it with a meaningful one to describe what this class does.

Reply:
ok. will do in a later pull request. please approve this one.

Reply:
Sure!